widgetastic_manageiq/__init__.py

```diff
@@ -865,21 +871,47 @@ def __init__(self, parent, title, *args, **kwargs):
     @property
     def fields(self):
         """Returns a list of the field names in the table (the left column)."""
-        return [row[0].text for row in self]
+        fields_names = []
+        for field in self:
+            if self.browser.get_attribute('class', field[0]):
+                fields_names.append(field[0].text)
+        return fields_names
 
     def get_field(self, field_name):
-        """Returns the table row of the field with this name.
-
+        """Returns the table row or list of elements for rowspam case
+                of the field with this name.
         Args:
             field_name: Name of the field (left column)
 
         Returns:
-            An instance of :py:class:`VanillaRow`
+            An instance of :py:class:`VanillaRow` or list of :py:class:`WebElement`
         """
+        rowspan_path = './tbody//td[contains(text(), {})]/following-sibling::td'.format(
+            quote(field_name))
         try:
-            return self.row((0, field_name))
+            rowspan_attribute = self.browser.get_attribute('rowspan', self.row((0, field_name))[0])
         except IndexError:
             raise NameError('Could not find field with name {!r}'.format(field_name))
+        if not rowspan_attribute:
+            return self.row((0, field_name))
+        else:
+            rowspan_image = 'i'
+            rowspan_attribute = 'class'
+            try:
+                rowspan_image_element = self.browser.element('{}/{}'.format(
```

Comment:
I guess you can replace both calls by the following:
./*[self::i or self::img] and defining rowspan_path as a parent